.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#10332D",
+    "titleBar.activeBackground": "#16473E",
+    "titleBar.activeForeground": "#F4FBFA"
+  }
+}
```

Comment:
No ruby sempre tem que pular uma linha no final :)

Reply:
Mas isso não pode ser commitado, pq é do vscode, vc pode criar um gitignore

https://stackoverflow.com/questions/42818891/gitignore-from-the-beginning-how-to-add-how-to-edit-on-rails/42818988